src/parsing/parse-tree.ts

```diff
@@ -24,7 +24,11 @@ declare module "../model/position" {
 
 export function positionOfParseTree(parseTree: ParseTree): Position | undefined {
     if(parseTree instanceof ParserRuleContext) {
-        return new Position(Point.ofTokenStart(parseTree.start), Point.ofTokenEnd(parseTree.stop));
+        const startToken = parseTree.start;
+        const stopToken = parseTree.stop;
+
+        if (stopToken)
```

Comment:
Perché if(stopToken) e non if(startToken)?

Reply:
Ho visto che `parseTree.start : Token`, mentre `parseTree.stop : Token | undefined` in antlr4ts
quindi in teoria startToken non dovrebbe essere undefined/null

Può andare bene oppure è buona norma/coerenza verificare anche startToken?

Reply:
No ok in tal caso però se stopToken è undefined ritornerei comunque la posizione del solo start token.